src/Accordion/accordion.js

```diff
@@ -28,17 +34,38 @@ class Accordion extends Component {
         this.renderItems = this.renderItems.bind(this);
     }
 
+    componentWillReceiveProps(nextProps) {
+        if (!isArraysEqual(nextProps.activeItems, this.state.activeItems)) {
```

Comment:
Rename this to `isArraysEqualShallow` to make sure there is no confusion about what is going on here?